streamlit/initial_strategy.py

```diff
@@ -73,7 +73,9 @@ def main():
 
     # simulation parameters
     random_seed = int(st.sidebar.number_input("Random seed", value=42))
-    strategy_name = st.sidebar.selectbox("Strategy", list(selection_strategies.keys()))
+    strategy_name = st.sidebar.selectbox(
```

Comment:
Should this still say "Strategy"? Ad should maybe the whole file also be renamed to `initial_recommender`?

Reply:
changed